src/test/java/org/openmrs/contrib/qaframework/page/AttachmentsPage.java

```diff
@@ -35,6 +42,22 @@ public void setFileUrl(String fileUrl) {
 		element.sendKeys(fileUrl);
 	}
 
+	public String createRandomFile() throws IOException {
+		File tempFile = File.createTempFile("file", ".pdf");
+		tempFile.deleteOnExit();
+		try (InputStream pdf = getClass().getClassLoader().getResourceAsStream("data/file.pdf")) {
+			try (Writer w = new FileWriter(tempFile)) {
```

Comment:
@Irenyak1 did you take a look at the naming convention of variables?

```suggestion
			try (Writer w = new FileWriter(tempFile)) {
```

Reply:
> 
@kdaud I personally don't see a problem in the variable names ie tempFile. Any pointers?

Reply:
`w`